conventions.md

```diff
@@ -0,0 +1,68 @@
+# Starlark Language Conventions
+
+This document describes non-normative but generally recommended conventions for
+Starlark.
+
+## API Documentation
+
+### Docstrings
+
+API documentation for a function should be provided in a [Python-style
```

Comment:
Are we matching the PEP 257 standard exactly or describing a new standard merely inspired by it?

Are we saying anything about "Args:" sections, etc., or is that out of scope for the conventions?

Reply:
Removed mention of PEP 257.